backend/settings.py

```diff
@@ -26,14 +26,14 @@ class Settings(BaseSettings):
     VECTOR_DB_CONFIG = os.getenv("VECTOR_DB_CONFIG", "")
     METADATA_STORE_CONFIG = os.getenv("METADATA_STORE_CONFIG", "")
     TFY_SERVICE_ROOT_PATH = os.getenv("TFY_SERVICE_ROOT_PATH", "")
-    JOB_FQN = os.getenv("JOB_FQN", "")
-    JOB_COMPONENT_NAME = os.getenv("JOB_COMPONENT_NAME", "")
+    JOB_FQN:ClassVar[str] = os.getenv("JOB_FQN", "")
+    JOB_COMPONENT_NAME:ClassVar[str] = os.getenv("JOB_COMPONENT_NAME", "")
```

Comment:
technically all of these can be class var, but why are we specifically changing these?

Reply:
These 2 were not present in env so need to annotate these, if not done they were throwing error:

PydanticUserError(pydantic.errors.PydanticUserError: A non-annotated attribute was detected: JOB_FQN = ''. All model fields require a type annotation; if JOB_FQN is not meant to be a field, you may be able to resolve this error by annotating it as a ClassVar or updating model_config['ignored_types'].